src/sales/order-update.md

```diff
@@ -2,12 +2,30 @@
 title: Updating an Order
 ---
 
-After a customer places an order, it is sometimes necessary to edit the order, place it on hold, or cancel it entirely. When you change an order, the original order is canceled and a new order is generated. You can, however, change the billing or shipping address without generating a new order.
+When helping a customer who has placed an order, the first thing you need to determine is the status of the order because the available options are different for a `pending` order than for one that is `processing`.
 
-![]({% link images/images/sales-order-edit.png %}){: .zoom}
-_Edit Order_
+## Pending Orders
 
-## To edit an order:
+After a customer places an order, but before the payment is processed, you can edit the order, place it on hold, or cancel it entirely. The button bar of a pending order lists the actions that can be applied.
+
+![]({% link images/images/order-button-bar-pending.png %}){: .zoom}
+_`Pending` Order Options_
+
+If you modify the substance of an order, the original order is canceled and a new order is generated. You can, however, change the billing or shipping address without generating a new order.
+
+## Processing Orders
+
+When the payment process begins, the status of the order changes to `processing`. Although it is no longer possible to change the substance of the order, the billing and shipping address can be edited. The order can no longer be canceled, although a credit memo can be issued for the full or partial amount of the order{% if "Default.EE-B2B" contains site.edition %}, or a return merchandise authorization (RMA) issued for product returns{% endif %}. To learn more, see [Processing an Order]({% link sales/order-processing.md %}).
+
+<!--{% if "Default.CE Only" contains site.edition %}-->
+![]({% link images/images/order-button-bar-processing.png %}){: .zoom}
+<!--{% endif %}-->
+<!--{% if "Default.EE-B2B Only" contains site.edition %}-->
+![]({% link images/images-ee/order-button-bar-processing.png %}){: .zoom}
+<!--{% endif %}-->
+_`Processing` Order Options_
+
+## Edit a pending order:
```

Comment:
```suggestion
## Edit a pending order
```

Linting rule soon to be implemented will not allow punctuation at the end of a heading, so better to address these now as we see them.